index.html

```diff
@@ -1194,55 +1194,48 @@ <h3>Contexts</h3>
 
       </section>
 
-      <section>
-        <h3>Identifiers</h3>
+      <section id="identifiers">
+        <h3>식별자</h3>
 
         <p>
-When expressing statements about a specific thing, such as a person, product,
-or organization, it is often useful to use some kind of identifier so that
-others can express statements about the same thing. This specification defines
-the optional <code>id</code> <a>property</a> for such identifiers. The
-<code>id</code> <a>property</a> is intended to unambiguously refer to an object,
-such as a person, product, or organization. Using the <code>id</code>
-<a>property</a> allows for the expression of statements about specific things in
-the <a>verifiable credential</a>.
+사람, 제품, 또는 조직과 같은 특정한 것에 대해 표현할때, 
+다른 사람들이 동일한 것을 표현할 수 있도록 식별자를 사용하는 것이 유용하다. 
+이 규격에서는 이러한 식별자들에 대한 선택적 <code>id</code> 속성을 정의한다. 
+<code>id</code> <a>속성</a>은 사람, 제품 또는 조직과 같은 객체를 명확하게 참조하기 위한 것이다. 
+<code>id</code> <a>속성</a>을 사용하면 
+<a>검증가능한 크리덴셜</a>을 통해 특정한 사물을 표현할 수 있게 된다.
         </p>
 
         <p>
-<em>If</em> the <code>id</code> <a>property</a> is present:
+<em>만약</em> <code>id</code> <a>속성</a>이 존재한다면:
         </p>
 
         <ul>
           <li>
-The <code>id</code> <a>property</a> MUST express an identifier that others are
-expected to use when expressing statements about a specific thing identified
-by that identifier.
+<code>id</code> <a>속성</a>을 표현할 때 반드시 다른 사용자들이 특정 사물을 표현할 때 사용할 것으로 예상되는 식별자로 표현해야 한다.
           </li>
           <li>
-The <code>id</code> <a>property</a> MUST NOT have more than one value.
+<code>id</code> <a>속성</a>은 반드시 하나의 값만 가져야 한다.
           </li>
           <li>
-The value of the <code>id</code> <a>property</a> MUST be a URI.
+<code>id</code> <a>속성</a>의 값은 반드시 URI 이어야 한다
           </li>
         </ul>
 
         <p class="note">
-Developers should remember that identifiers might be harmful in scenarios
-where pseudonymity is required. Developers are encouraged to read Section
-<a href="#identifier-based-correlation"></a> carefully when considering such
-scenarios. There are also other types of correlation mechanisms documented in
-Section <a href="#privacy-considerations"></a> that create privacy concerns.
-Where privacy is a strong consideration, the <code>id</code> <a>property</a>
-MAY be omitted.
+개발자들은 익명성이 요구되는 시나리오에서 식별자가 유해할 수 있다는 것을 기억해야 한다. 
+그러한 시나리오를 고려할 때 <a href="#identifier-based-correlation"></a> 부분을 주의깊게 읽을 것을 권장한다. 
+프라이버시 문제를 만들어 내는 다른 유형의 연관메커니즘문서도 
+<a href="#privacy-considerations"></a>에서 다루어 진다. 
```

Comment:
```suggestion
<a href="#privacy-considerations"></a>에서 다룬다. 
```
번역체를 살짝 피해보면 어떨까 싶습니다.